app/models/account.rb

```diff
@@ -36,6 +36,7 @@
 #  followers_count         :integer          default(0), not null
 #  following_count         :integer          default(0), not null
 #  last_webfingered_at     :datetime
+#  activitystreams2_url    :string
```

Comment:
If I understand correctly AS2 could also be used for OStatus, so this might be confusing.